azurerm/data_source_container_orchestrator_versions.go

```diff
@@ -0,0 +1,87 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmContainerOrchestratorVersions() *schema.Resource {
+	return &schema.Resource{
+		Read: dataArmSourceContainerOrchestratorVersionsRead,
+
+		Schema: map[string]*schema.Schema{
+			"location": locationSchema(),
+			"orchestrator_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "Kubernetes",
+			},
```

Comment:
as can remove the field `orchestrator_type` since we're only going to support Kubernetes at this time

Reply:
Sure thing. I'm on it.